src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesAEAD.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// ThF:\src\corefx\src\System.Security.Cryptography.Algorithms\src\System\Security\Cryptography\AesAEAD.Windows.cse .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This is the only place I see this particular error, but: paste error inside the license header